images/macos/toolsets/toolset-11.0.json

```diff
@@ -88,5 +88,11 @@
                 "1.15.*"
             ]
         }
+    ],
+    "pipx": [
+        {
+            "package": "yamllint",
+            "cmd": "yamllint"
```

Comment:
Why do we add cmd field? Looks like it is not used anywhere

Reply:
Also could we iterate by this JSON in tests?

Reply:
Rework tests